web/lib/util/embed.ts

```diff
@@ -0,0 +1,16 @@
+import { useEffect, useState } from 'react'
+
+export function inIframe() {
```

Comment:
Might not need to export if you should always call it from an effect clientside, in which case you would just use the hook below.